modules/phase_field/test/tests/actions/gpm_kernel.i

```diff
@@ -0,0 +1,200 @@
+      Fj_w = 'rhob rhom'
+
+      gamma_gr = gamma
+      gr_mob_name = L
+      kappa_gr = kappa
+      Fj_gr = 'omegab omegam'
+      en_barr_gr = mu
+
+      additional_ops = 'phi'
+      gamma_grxop = gamma
+      op_mob_name = L_phi
+      kappa_op = kappa
+      Fj_op = 'omegab omegam'
+      en_barr_op = mu
+    [../]
+  [../]
+[]
+
+[AuxKernels]
+  [./bnds_aux]
+    type = BndsCalcAux
+    variable = bnds
+  [../]
+[]
+
+[Materials]
+  #REFERENCES
+  [./constants]
+    type = GenericConstantMaterial
+    prop_names =  'Va      cb_eq cm_eq kb   km  mu  gamma L      L_phi  kappa  kB'
+    prop_values = '0.04092 1.0   1e-5  1400 140 1.5 1.5   5.3e+3 2.3e+4 295.85 8.6173324e-5'
+  [../]
+  #SWITCHING FUNCTIONS
+  [./switchb]
+    type = SwitchingFunctionMultiPhaseMaterial
+    h_name = hb
+    all_etas = 'phi eta0'
+    phase_etas = 'phi'
+  [../]
+  [./switchm]
+    type = SwitchingFunctionMultiPhaseMaterial
+    h_name = hm
+    all_etas = 'phi eta0'
+    phase_etas = 'eta0'
+  [../]
+  [./omegab]
+    type = DerivativeParsedMaterial
+    f_name = omegab
+    args = 'w phi'
+    material_property_names = 'Va kb cb_eq'
+    function = '-0.5*w^2/Va^2/kb - w/Va*cb_eq'
+    derivative_order = 2
+  [../]
+  [./omegam]
+    type = DerivativeParsedMaterial
+    f_name = omegam
+    args = 'w eta0'
+    material_property_names = 'Va km cm_eq'
+    function = '-0.5*w^2/Va^2/km - w/Va*cm_eq'
+    derivative_order = 2
+  [../]
+  [./chi]
+    type = DerivativeParsedMaterial
+    f_name = chi
+    args = 'w'
+    material_property_names = 'Va hb hm kb km'
+    function = '(hm/km + hb/kb)/Va^2'
+    derivative_order = 2
+  [../]
+  #DENSITIES/CONCENTRATION
+  [./rhob]
+    type = DerivativeParsedMaterial
+    f_name = rhob
+    args = 'w'
+    material_property_names = 'Va kb cb_eq'
+    function = 'w/Va^2/kb + cb_eq/Va'
+    derivative_order = 1
+  [../]
+  [./rhom]
+    type = DerivativeParsedMaterial
+    f_name = rhom
+    args = 'w eta0'
+    material_property_names = 'Va km cm_eq(eta0)'
+    function = 'w/Va^2/km + cm_eq/Va'
+    derivative_order = 1
+  [../]
+  [./concentration]
+    type = ParsedMaterial
+    f_name = c
+    material_property_names = 'rhom hm rhob hb Va'
+    function = 'Va*(hm*rhom + hb*rhob)'
+    outputs = exodus
+  [../]
+  [./diffusivity]
+    type = DerivativeParsedMaterial
+    material_property_names = 'chi kB'
+    constant_names = 'T Em D0'
+    constant_expressions = '1400 2.4 1.25e2'
+    f_name = D
+    function = 'chi*D0*exp(-Em/kB/T)'
+  [../]
+[]
+
+[Preconditioning]
+  [./SMP]
+    type = SMP
+    full = true
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  scheme = bdf2
+  solve_type = NEWTON
+  #petsc_options_iname = '-pc_type -pc_factor_mat_solver_package'
+  #petsc_options_value = ' lu       mumps'
+  petsc_options_iname = '-pc_type -sub_pc_type -pc_asm_overlap -ksp_gmres_restart -sub_ksp_type'
+  petsc_options_value = ' asm      lu           1               31                 preonly'
```

Comment:
does this really need all these PETSc options? Please remove commented out stuff.